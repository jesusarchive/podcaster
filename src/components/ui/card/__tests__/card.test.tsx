import { render } from '@testing-library/react';
import React from 'react';

import Card from '../card';

describe('Card', () => {
  it('should render the card component', () => {
    const { container } = render(<Card>Card</Card>);

    expect(container).toMatchSnapshot();
  });
});
