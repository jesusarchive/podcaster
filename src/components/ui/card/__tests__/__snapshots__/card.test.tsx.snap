// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render the card component 1`] = `
<div>
  <div
    class="card"
  >
    Card
  </div>
</div>
`;
